services/checkin/controller/controller.go

```diff
@@ -102,20 +102,20 @@ func UpdateUserCheckin(w http.ResponseWriter, r *http.Request) {
 	err := service.UpdateUserCheckin(user_checkin.ID, user_checkin)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.DatabaseError(err.Error(), "Could not update user check-in information."))
 	}
 
 	updated_checkin, err := service.GetUserCheckin(user_checkin.ID)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.DatabaseError(err.Error(), "Could not fetch updated check-in information."))
 	}
 
 	if updated_checkin.Override {
 		err = service.AddAttendeeRole(updated_checkin.ID)
 
 		if err != nil {
-			panic(errors.UnprocessableError(err.Error()))
+			panic(errors.InternalError(err.Error(), "Could not add attendee role.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.